src/plugins/intel_npu/src/plugin/npuw/accuracy/comparator.cpp

```diff
@@ -13,30 +13,47 @@
 ov::npuw::metrics::NRMSE::NRMSE(double threshold) : m_threshold(threshold) {}
 
 bool ov::npuw::metrics::NRMSE::operator()(const ov::SoPtr<ov::ITensor>& actual,
-                                          const ov::SoPtr<ov::ITensor>& reference) const {
-    NPUW_ASSERT(actual->is_continuous());
-    NPUW_ASSERT(reference->is_continuous());
+                                          const ov::SoPtr<ov::ITensor>& reference,
+                                          double* result) const {
```

Comment:
This `bool operator() (.., double *result);` thing looks weird. Just make a proper return type like `Result`.

For those who were only checking true/false, that type could have `operator bool()`.